docs/arcaflow/workflows/matrix.md

```diff
@@ -0,0 +1,7 @@
+# Workflow and Engine Compatibility Matrix
+
+The workflow schema version uses the [semantic versioning 2.0.0 specification](https://semver.org/). In a workflow file definition, the schema version is prepended with a `v`. For example, `0.1.0` is written as `v0.1.0`.
```

Comment:
Do you want to explicitly recommend always creating a new workflow with the latest version? 😸

Reply:
Yeah, we need to be careful. Because the documentation's version needs to match what the documentation itself is recommending.